tests/app/main/views/test_user_profile.py

```diff
@@ -263,7 +263,7 @@ def test_non_gov_user_cannot_see_change_email_link(
 ):
     client_request.login(api_nongov_user_active)
     page = client_request.get('main.user_profile')
-    assert '<a href="/user-profile/email">' not in str(page)
+    assert '<a class="govuk-link govuk-link--no-visited-state" href="/user-profile/email">' not in str(page)
```

Comment:
Could write it like this so that the test doesn't need updating every time we change the class, but I'll leave that decision up to you
```suggestion
    assert not page.find('a', {'href': url_for('main.user_profile_email')})
```